test/wappsto_test.py

```diff
@@ -222,8 +236,9 @@ def test_receive_thread_method(self, id, verb, callback_exists, expected):
             args, kwargs = self.service.socket.sending_queue.put.call_args
 
         # Assert
-        assert int(args[0].rpc_id) == id
-        assert args[0].msg_id == expected
+        assert args[0].rpc_id == expected_rpc_id
+        assert args[0].msg_id == expected_msg_id
+        assert args[0].trace_id == expected_trace_id
```

Comment:
You need to also mock the method that sends the real trace request to "tracer.iot.seluxit.com" to make sure that it is called and with the correct trace id.